apps/prairielearn/src/pages/instructorInstanceAdminSettings/instructorInstanceAdminSettings.sql

```diff
@@ -6,13 +6,3 @@ FROM
 WHERE
   ci.course_id = $course_id
   AND ci.deleted_at IS NULL;
-
--- BLOCK select_course_instance_id_from_uuid
-SELECT
-  ci.id AS course_instance_id
-FROM
-  course_instances AS ci
-WHERE
-  ci.uuid = $uuid
-  AND ci.course_id = $course_id
-  AND ci.deleted_at IS NULL;
```

Comment:
I put this into a model function, so I updated other places to use the new model function even though they were unrelated.

Reply:
This would be pretty straightforward to move to a separate PR too to clean up the diff.

Reply:
done: #12138